From f7d77f4c74337188b7e5da589c0a0f95c366856d Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:23:48 -0400
Subject: [redhat] kabi: Adding symbol invalidate_partition (block/genhd.c)

Message-id: <d232eb7358f49975755d8bff07e9f5472ade7721.1410510773.git.poros@redhat.com>
Patchwork-id: 96106
O-Subject: [PATCH RHEL7.1 108/233] kabi: Adding symbol invalidate_partition (block/genhd.c)
Bugzilla: 1062361
RH-Acked-by: Jeff Moyer <jmoyer@redhat.com>

Bugzilla: 1062361
    https://bugzilla.redhat.com/show_bug.cgi?id=1062361

    Description:
    ============
    PowerPath needs to update the partition table for a device after
    certain operations such as migrating data from one lun to another.
    We use invalidate_partition for this purpose.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/invalidate_partition
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/invalidate_partition
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/invalidate_partition

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/invalidate_partition b/redhat/kabi/kabi-module/kabi_ppc64/invalidate_partition
new file mode 100644
index 0000000..04d0ac5
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/invalidate_partition
@@ -0,0 +1,2 @@
+#1-
+0x5bb092bb invalidate_partition vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/invalidate_partition b/redhat/kabi/kabi-module/kabi_s390x/invalidate_partition
new file mode 100644
index 0000000..1fb6270
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/invalidate_partition
@@ -0,0 +1,2 @@
+#1-
+0x2ec53334 invalidate_partition vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/invalidate_partition b/redhat/kabi/kabi-module/kabi_x86_64/invalidate_partition
new file mode 100644
index 0000000..e79c471
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/invalidate_partition
@@ -0,0 +1,2 @@
+#1-
+0xcfda4de5 invalidate_partition vmlinux EXPORT_SYMBOL
-- 
1.7.1